graphix/parameter.py

```diff
@@ -0,0 +1,921 @@
+"""Parameter class
+
+Parameter object acts as a placeholder of measurement angles and
+allows the manipulation of the measurement pattern without specific
+value assignment.
+
+"""
+
+from __future__ import annotations
+
+import cmath
+import functools
+import numbers
+from enum import Enum
+
+import numpy as np
+
+
+class Expression:
+    """Expression with parameters."""
+
+    def __mul__(self, other) -> Expression | complex | type(NotImplemented):
```

Comment:
Please do not specify `NotImplemented` type, as this value itself is never returned from `__mul__`.

It's just a marker for internals.